### PR TITLE
Restore go-live setting service limit to 10k. 

### PR DESCRIPTION
**Pull request by whabanks**
# Summary | Résumé
Reverting work done in [this PR](https://github.com/cds-snc/notification-admin/pull/935) to ensure that when a service goes live, their message limit is properly set to 10k. When the service is no longer live, but restricted, the limit is restored to the default of 50.

# Test instructions | Instructions pour tester la modification
Create a service and change the daily message limit. Go live with the service. Note that the daily message limit is set to 10k.
Turn off live, and note that the message limit is set to 50.

### CODE REVIEW

File: app/main/views/service_settings.py [hunk 1]
```diff
@@ -368,18 +368,11 @@ def service_switch_live(service_id):
...
 
-            if current_service.sms_daily_limit != current_app.config["DEFAULT_SMS_DAILY_LIMIT"]:
-                sms_daily_limit = current_service.sms_daily_limit
-            else:
-                sms_daily_limit = current_app.config["DEFAULT_LIVE_SMS_DAILY_LIMIT"]
+        if live:
+            message_limit = current_app.config["DEFAULT_LIVE_SERVICE_LIMIT"]
 
-            flash(_("An email has been sent to service users"), "default_with_tick")
+        flash(_("An email has been sent to service users"), "default_with_tick")
```
Comment (andrewleith) [inline app/main/views/service_settings.py:376]: Makes much more sense to me! 👍

File: tests/app/main/views/test_service_settings.py [hunk 2]
```diff
@@ -622,10 +622,10 @@ def test_show_restricted_service(
 @pytest.mark.parametrize(
     "current_limit, expected_limit, current_sms_limit, expected_sms_limit",
     [
-        (42, 42, 33, 33),
+        (42, 10_000, 33, 50),
```
Comment (andrewleith) [inline tests/app/main/views/test_service_settings.py:625]: Isn't the default live sms limit supposed to be 1000? Maybe I'm [looking in the wrong place](https://github.com/cds-snc/notification-admin/blob/main/app/config.py#L66).

**Review by andrewleith — COMMENTED** [review]
A question below on the default number of SMS for a live service.

File: tests/app/main/views/test_service_settings.py (hunk 2 above)
Comment (whabanks) [reply tests/app/main/views/test_service_settings.py:625]: That looks more correct to me, I'll make the adjustment.

**Review by andrewleith — APPROVED** [review]
Thats better! Not sure why deploying the pr environment is failing, ill take a look at that now.

**Pull request merged**